docker-compose.test.yml

```
@@ -0,0 +1,140 @@
+      test: [ "CMD", "redis-cli", "--raw", "incr", "ping" ]
+      interval: 10s
+      timeout: 10s
+      retries: 2
+      start_period: 10s
+    networks:
+      - temporal-network
+
+networks:
+  temporal-network:
+    driver: bridge
+    name: temporal-network
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)